packages/typespec-test/test/contentsafety_modular/generated/typespec-ts/src/ContentSafetyClient.ts

```diff
@@ -57,18 +59,18 @@ export class ContentSafetyClient {
 
   /** A sync API for harmful content analysis for text. Currently, we support four categories: Hate, SelfHarm, Sexual, Violence. */
   analyzeText(
-    text: string,
+    body: AnalyzeTextOptions,
     options: AnalyzeTextRequestOptions = { requestOptions: {} }
   ): Promise<AnalyzeTextResult> {
-    return analyzeText(this._client, text, options);
+    return analyzeText(this._client, body, options);
   }
 
   /** A sync API for harmful content analysis for image. Currently, we support four categories: Hate, SelfHarm, Sexual, Violence. */
   analyzeImage(
-    image: ImageData,
+    body: AnalyzeImageOptions,
```

Comment:
Can we have a better name instead of 'body' here?  I think that is consistent with HLC.
    analyzeTextOptions:  AnalyzeTextOptions

Reply:
This is their body name and AnalyzeImageOptions is their model name. I checked with them, they won't like to change it.

Reply:
Yeah, I was thinking the same as Renhe, but it looks like this is following what is defined in the spec. Python DPG has body as the parameter name as well

Reply:
as Jeff confirmed with Brian, we should not done anything tricky to manipulate the name in codegen side.